samples/charts/area/line-boundaries.html

```diff
@@ -18,14 +18,14 @@
 		<div class="wrapper col-2"><canvas id="chart-3"></canvas></div>
 
 		<div class="toolbar">
-			<button onclick="toggleSmooth(this)">Smooth</button>
-			<button onclick="randomize(this)">Randomize</button>
+			<button id="toggleSmooth">Smooth</button>
+			<button id="randomize">Randomize</button>
 		</div>
 	</div>
 
 	<script>
 		var presets = window.chartColors;
-		var utils = Samples.utils;
+		var utils = window.Samples.utils;
```

Comment:
`Samples` needs to be added to the eslintrc globals, then we can remove `window.`

I didn't comment other similar changes, but it needs to be fixed in the whole PR.